tests/models_tests/models_test.go

```diff
@@ -678,6 +678,12 @@ func TestGetAllCohortDefinitionsAndStatsOrderBySizeDesc(t *testing.T) {
 		t.Errorf("Expected teamProject '%s' to have one cohort, but found %d",
 			testTeamProject, len(allowedCohortDefinitions))
 	}
+	testTeamProject = "teamprojectX"
+	allowedCohortDefinitions, _ = cohortDefinitionModel.GetAllCohortDefinitionsAndStatsOrderBySizeDesc(testSourceId, testTeamProject)
+	if len(allowedCohortDefinitions) != 2 {
+		t.Errorf("Expected teamProject '%s' to have 2 cohorts, but found %d",
+			testTeamProject, len(allowedCohortDefinitions))
+	}
```

Comment:
check just added for completion. This PR's changes are really in the controller layer.